tests/cpp/runtime/parsing-functions-test.cpp

```diff
@@ -0,0 +1,218 @@
+#include <gtest/gtest.h>
+#include <iostream>
+#include <fstream>
+#include "runtime/parsing_functions.h"
+
+/*
+ * Tests for helper functions
+ */
+
+TEST(parsing_functions_test, test_clear_quotes) {
+  ASSERT_STREQ(
+    string("Single quotes string").c_str(),
+    clearQuotes(string("\'Single quotes string\'")).c_str()
+  );
+  ASSERT_STREQ(
+    string("Double quotes string").c_str(),
+    clearQuotes(string("\"Double quotes string\"")).c_str()
+  );
+}
+
+TEST(parsing_functions_test, test_clear_extra_spaces) {
+  ASSERT_STREQ(
+    string("This is string without extra spaces").c_str(),
+    clearExtraSpaces(string("This           is     string   without     extra spaces")).c_str()
+  );
+}
+
+
+TEST(parsing_functions_test, test_tolower) {
```

Comment:
This test will be redundant if you'll use already available `f$strtolower` function.